.travis.yml

```diff
@@ -46,6 +46,7 @@ jobs:
 
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
+    - env: EMBER_TRY_SCENARIO=ember-3.13
```

Comment:
We don't use Travis anymore. We should likely delete this file.

Reply:
Done, I've removed this file.